handler/release_handler.go

```diff
@@ -204,3 +206,13 @@ func updateRelease(client *github.Client, release *github.RepositoryRelease, own
 	_, _, err := client.Repositories.EditRelease(context.Background(), owner, repo, *release.ID, release)
 	return err
 }
+
+func includeCommits(cm github.RepositoryCommit, previous, current time.Time) bool {
```

Comment:
I think this should be called `includeCommit` given that you are working on a single item at a time?

Reply:
Agree, will make that change.

Reply:
Done